src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.module.css

```diff
@@ -137,21 +137,18 @@
   position: relative;
 }
 
-.leftBottomImageWrapper.solo {
-  margin-top: 40px;
+.leftBottomImageWrapper.paired {
+  margin-top: -30px;
+  min-width: 100%;
```

Comment:
Is this min-width necessary? Setting a min-width of 100% is fairly unusual, since it means you're forcing the element to be as large as or larger than its parent (minus additional padding + margin).